app.rb

```diff
@@ -0,0 +1,47 @@
+require 'sinatra'
+require "sinatra/reloader"
+require_relative 'lib/database_connection'
+require_relative 'lib/user_repository'
+require_relative 'lib/chitter_repository'
+
+DatabaseConnection.connect
+
+class Application < Sinatra::Base 
+    
+    configure :development do
+        register Sinatra::Reloader
+        also_reload 'lib/user_repository'
+        also_reload 'lib_chitter_repository'
+    end
+    get '/peeps' do
+       repi = ChitterRepository.new
+       repos = UserRepository.new
+       a_user = repos.all
+       peepi = repi.all
+       @contents = []
+       @user_name = []
+       a_user.each do |u|
```

Comment:
using more descriptive names for variables is essential for others to work out what the code is for